### PR TITLE
plafrom_name consistency.adoc

### PR DESCRIPTION
**Pull request by vturpin**
* Reference to plaform name

* correction on platform exemplar in the global attribute

Proponents: 
Moderator: @OceanGlidersCommunity/format-mantainers

# Type of PR

- [ ] Typo without possible change of interpretation of the related text.
- [X] Fix of some error, inconsistency, unforeseen limitation.
- [ ] Style that only affects visually the compiled document
- [ ] Addition that does not require change in the current structure.
- [ ] Enhancement that require changes to improve the format.

# Related Issues

# Dates when it got review approvals

# Release checklist

- [ ] Approved by at least two members of the committee?
- [ ] There were modifications after the review approvals? If so, please
      ask reviewers to update their review.
- [ ] Proponents and moderador should explicitly agree that it is ready to
      to merge.
- [ ] The moderador is the one in charge to actually merge or close this PR
      according to the final decision.

# For maintainers

- [ ] Update the moderator with a volunteer from the committee. It would be
      best to have one single moderator to guide and help this PR to move
      forward. It is OK to update the moderador pass it to another one.
- [ ] Confirm that the associated branch was deleted after the merging.
- [ ] Wrap-up and close the related issues.

# Comments

### CODE REVIEW

File: OG_Format.adoc [hunk 1]
```diff
@@ -141,9 +141,10 @@ data_mode_vocabulary = “”;
 |mandatory |
 ex.:
 
-* sverdrup_20200512T001245_delayed
-* SL287_20180715T012451_delayed
-* p202_20150923T150451_R
+* unit1032_20230512T001245_delayed
```
Comment (emmerbodc) [inline OG_Format.adoc:144]: include underscore after unit

File: OG_Format.adoc [hunk 2]
```diff
@@ -401,9 +402,7 @@ In this following section, two options, “encapsulate variable” and “indivi
 
 Value: <platform_name>_<start_date>
 
-Where <platform_name> refers to the name of the glider, <start_date> refers to the deployment start UTC date under iso 8601. *Ex : eltanin_20210909T1605*
-
-If the glider has no <platform_name> use <platform_serial_number> instead to create the TRAJECTORY. *Ex.: sp042_20210218T2325*
+Where <platform_name> is defined below , <start_date> refers to the deployment start UTC date under iso 8601. *Ex : sp041_20210909T1605*
```
Comment (emmerbodc) [inline OG_Format.adoc:405]: to include seconds on this example

Comment (vturpin) [reply OG_Format.adoc:405]: solved in next commit

**Comment by vturpin** [comment]
@emmerbodc can you review the changes made after your comments before merging if ok ?

File: OG_Format.adoc [hunk 3]
```diff
@@ -440,7 +439,7 @@ If the glider has no <platform_name> use <platform_serial_number> instead to cre
 * long_name = “unique identifying name of glider”;
  |mandatory
 
- The  platform name should be constructed from the manufacturer prefix and the platform serial number e.g. "sea001" (seaglider), unit001 (slocum), sea001 (seaexplorer), sp001 (spray). Where the serial number of the platform is not known, a local nickname can be used e.g. "orca", "sverdrup", "ammonite".
+ The  platform name should be constructed from the manufacturer prefix and the platform serial number e.g. "sea001" (seaglider), unit_001 (slocum), sea001 (seaexplorer), sp001 (spray). Where the serial number of the platform is not known, a local nickname can be used e.g. "orca", "sverdrup", "ammonite".
```
Comment (emmerbodc) [inline OG_Format.adoc:442]: can you update the seaglider example  - e.g. "sg638" (seaglider)

Comment (castelao) [inline OG_Format.adoc:442]: Maybe use "se" for sea explorer and "sg" for sea glider to keep consistency between the two letters.

Comment (callumrollo) [reply OG_Format.adoc:442]: alseamar refer to their seaexplorers as sea001, including in all filenames and documentation. I recommend keeping to that.

Comment (vturpin) [reply OG_Format.adoc]: resolved

**Review by emmerbodc — APPROVED** [review]

**Review by glidermann — APPROVED** [review]

**Pull request merged**